composer.json

```diff
@@ -23,6 +23,9 @@
     "require": {
         "php": ">=5.2.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "4.*"
```

Comment:
i assume that's to assert that the behaviour is the same as the actual php implementation